views/clockviews/shared.html

```diff
@@ -30,19 +30,41 @@
     <div class="inner">
       <div id="trend">
         <div id="bgnow"></div>
-        <div id="arrowDiv"><img id="arrow" src=""/></div>
+        <div id="arrowDiv"><img id="arrow" src="" alt="arrow"/></div>
       </div>
       <div id="clock"></div>
       <div id="staleTime"></div>
     </div>
   </main>
-  
-  <script src="/api/v1/status.js"></script>
-  <script src="<%= locals.bundle %>/js/bundle.clock.js?v=<%= locals.cachebuster %>"></script>
-  
+  <script src="/<%= locals.bundle %>/js/bundle.clock.js?v=<%= locals.cachebuster %>"></script>
```

Comment:
locals.bundle is `/bundle` by default, so I get error:
![image](https://user-images.githubusercontent.com/6500826/63356185-a2aff880-c367-11e9-9601-51c31926200d.png)
remove `/` in front of it to make it work